tests/cache/sdkman.bats

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bats
+
+setup() {
```

Comment:
The `setup` method should only be used to prepare necessary steps. Imo installing `sdkman` should be done inside a test.